question28/js/register.js

```diff
@@ -63,7 +63,15 @@ checkbox.addEventListener("change", () => {
 });
 
 submitButton.addEventListener('click',() => {
-  const userData = Object.fromEntries([...new FormData(form)]);
-  localStorage.setItem('userData', JSON.stringify(userData)); 
+  const registeredUserData = JSON.parse(localStorage.getItem('userData'));
+  const inputUserData = Object.fromEntries([...new FormData(form)]);
```

Comment:
[ comment ]
It was very smartly written and I learned a lot!
I didn't know about new FormData().
Thank you!

Reply:
comment
I've never heard of `new FormData ()` either! Learning, thank you!